bpf/lib/conntrack.h

```diff
@@ -369,6 +396,31 @@ ipv6_ct_tuple_swap_ports(struct ipv6_ct_tuple *tuple)
 	tuple->dport = tmp;
 }
 
+static __always_inline int
+ct_extract_ports6(struct __ctx_buff *ctx, int off,
+		  enum ct_dir dir __maybe_unused, struct ipv6_ct_tuple *tuple,
+		  bool *has_l4_header __maybe_unused)
+{
+	switch (tuple->nexthdr) {
```

Comment:
This needs to handle SCTP now as well, correct?

Reply:
Sounds correct, I added it.